xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxFFmpeg.cpp

```diff
@@ -725,6 +725,8 @@ AVDictionary* CDVDDemuxFFmpeg::GetFFMpegOptionsFromInput()
   CURL url = m_pInput->GetURL();
   AVDictionary* options = nullptr;
 
+  av_dict_set(&options, "protocol_whitelist", "file,http,https,tcp,tls,crypto", 0);
```

Comment:
don't you think there is a reason why ffmpeg implemented this option? blindly bypassing it for all cases does not look right.

Reply:
You should be able to have any protocols in a local m3u8 file. So I guess it should be all if not an internet stream and left as default otherwise.